src/plugins/console/public/application/hooks/use_data_init/use_data_init.ts

```diff
@@ -58,7 +59,7 @@ export const useDataInit = () => {
           const newObject = await objectStorageClient.text.create({
             createdAt: Date.now(),
             updatedAt: Date.now(),
-            text: '',
+            text: DEFAULT_INPUT_VALUE,
```

Comment:
@zhongnansu I am using `DEFAULT_INPUT_VALUE` here to avoid blank query export initially, should I replace default string with this variable here?

something like:

```js
 const newObject = await objectStorageClient.text.create({
            createdAt: Date.now(),
            updatedAt: Date.now(),
            text: `GET _search
              {
                "query": {
                  "match_all": {}
                }
              }`,
          });
```

Reply:
A const is good to me. The point was to avoid changing access level of any code in `legacy` folder. Thanks